local_utils.py

```diff
@@ -0,0 +1,55 @@
+
+import os
+from pathlib import Path
+from google.cloud import storage
+import platform
+import socket
+import psutil
+import logging
+import GPUtil
+import sys
+import tensorflow as tf
+import os
+os.environ['SM_FRAMEWORK'] = 'tf.keras'
+import segmentation_models
+
+
+def folder_has_files(directory, id):
```

Comment:
i think it's better named as a counterpart to `remote_folder_exists` in gcp_utils, so perhaps we name it `local_folder_exists` here?

Reply:
unless the directory in question might actually exist while being empty, and thus checking for files really is the key, but i don't think the local directory can exist AND be empty

Reply:
It does create a folder with no files, that is why I'm checking if there are files inside the folder. Because it makes a folder, and them copies stuff there (and maybe copies nothing):

```
def copy_folder_locally_if_missing(folder_remote_source, local_folder_dir):
    if not os.path.exists(local_folder_dir.as_posix()):
        local_folder_dir.mkdir(parents=True, exist_ok=True)
        os.system("gsutil -m cp -r '{}' '{}'".format(folder_remote_source, local_folder_dir.as_posix()))
```

Reply:
I'm going to go with `local_folder_has_files` for the name

Reply:
awesome, thx for the explanation!